.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,56 @@
+name: Bug report
+description: Report a bug in Godot
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      - Read our [CONTRIBUTING.md guide](CONTRIBUTING.md#reporting-bugs) on reporting bugs.
+      - Write a descriptive issue title above.
+      - Search [open](https://github.com/godotengine/godot/issues) and [closed](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported.
+      - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html).
+
+- type: input
+  attributes:
+    label: Godot version
+    description: >
+      Specify the Git commit hash if using a development or non-official build.
+      If you use a custom build, please test if your issue is reproducible in official builds too.
+    placeholder: Example - "3.3.stable" or "4.0.dev (3041becc6)"
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: System information
+    description: >
+      Specify the OS version, and when relevant hardware information.
+      For graphics-related issues, specify the GPU model, driver version, and the rendering backend (GLES2, GLES3, Vulkan).
+    placeholder: Example - "Windows 10, Intel HD Graphics 620 (27.20.100.9616), GLES3"
```

Comment:
This ended up being too long for the default input field width:
![Screenshot_20210608_161017](https://user-images.githubusercontent.com/4701338/121200462-0b859f00-c874-11eb-849f-f67b6556745b.png)

Maybe the "Example - " part and quotes can be removed, IMO it's explicit enough that those are placeholders and thus examples.

Maybe GLES3 can also come before the GPU and driver version, so worse case if there's cropping it's the driver details:

```suggestion
    placeholder: Windows 10, GLES3, Intel HD Graphics 620 (27.20.100.9616)
```